docs/kafka/manage-cluster/README.adoc

```diff
@@ -0,0 +1,250 @@
+:nodejs-url-kafka: kafka/nodejs-kafka/README.adoc
+:getting-started-rhoas-cli-url-kafka: kafka/rhoas-cli-getting-started-kafka/README.adoc
+:topic-config-url-kafka: kafka/topic-configuration-kafka/README.adoc
+:consumer-config-url-kafka: kafka/consumer-configuration-kafka/README.adoc
+:access-mgmt-url-kafka: kafka/access-mgmt-kafka/README.adoc
+:metrics-monitoring-url-kafka: kafka/metrics-monitoring-kafka/README.adoc
+:service-binding-url-kafka: kafka/service-binding-kafka/README.adoc
+:message-browsing-url-kafka: kafka/message-browsing-kafka/README.adoc
+
+//OpenShift Service Registry
+:product-long-registry: OpenShift Service Registry
+:product-registry: Service Registry
+:registry: Service Registry
+:product-version-registry: 1
+:service-url-registry: https://console.redhat.com/application-services/service-registry/
+:getting-started-url-registry: registry/getting-started-registry/README.adoc
+:quarkus-url-registry: registry/quarkus-registry/README.adoc
+:getting-started-rhoas-cli-url-registry: registry/rhoas-cli-getting-started-registry/README.adoc
+:access-mgmt-url-registry: registry/access-mgmt-registry/README.adoc
+:content-rules-registry: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1/guide/9b0fdf14-f0d6-4d7f-8637-3ac9e2069817[Supported Service Registry content and rules]
+:service-binding-url-registry: registry/service-binding-registry/README.adoc
+
+//OpenShift Connectors
+:connectors: Connectors
+:product-long-connectors: OpenShift Connectors
+:product-connectors: Connectors
+:product-version-connectors: 1
+:service-url-connectors: https://console.redhat.com/application-services/connectors
+:getting-started-url-connectors: connectors/getting-started-connectors/README.adoc
+:getting-started-rhoas-cli-url-connectors: connectors/rhoas-cli-getting-started-connectors/README.adoc
+:addon-url-connectors: https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/15a79de0-8827-4bf1-b445-8e3b3eef7b01
+
+
+//OpenShift API Designer
+:product-long-api-designer: OpenShift API Designer
+:product-api-designer: API Designer
+:product-version-api-designer: 1
+:service-url-api-designer: https://console.redhat.com/application-services/api-designer/
+:getting-started-url-api-designer: api-designer/getting-started-api-designer/README.adoc
+
+//OpenShift API Management
+:product-long-api-management: OpenShift API Management
+:product-api-management: API Management
+:product-version-api-management: 1
+:service-url-api-management: https://console.redhat.com/application-services/api-management/
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-managing-cluster"]
+= Managing {product-long-kafka} on a ROSA cluster
+ifdef::context[:parent-context: {context}]
+:context: deploying-cluster
+
+// Purpose statement for the assembly
+[role="_abstract"]
+
+As an organization administrator, you can deploy {product-long-kafka} within your own controlled environment on your own ROSA cluster.
```

Comment:
```suggestion
As an organization administrator, you can deploy {product-long-kafka} within your own controlled environment on your own {rosa-name} (ROSA) cluster.
```

Reply:
Suggest spelling out first occurrence of ROSA acronym using `{rosa-name}` attribute. 

Note that this attribute is currently set to `OpenShift Service for AWS` in upstream and downstream repos, but should be `OpenShift Service on AWS`. See https://www.redhat.com/en/technologies/cloud-computing/openshift/aws

Reply:
Thanks @smccarthy-ie for the great catch. Will amend it in both repos.